extension/package.json

```diff
@@ -4,11 +4,11 @@
   "description": "Parity Web Extension",
   "scripts": {
     "postinstall": "npm link ../signer/web",
-    "start": "node scripts/dev",
-    "build": "NODE_ENV=production node scripts/build",
+    "start": "npm run devBuild -- -w --port 3030",
```

Comment:
So `--port 3030` is superflous. I think previous setup was actually able to do Hot Reload for extension. But I guess refreshing is also fine.